lib/wallaby/experimental/chrome.ex

```diff
@@ -210,6 +210,21 @@ defmodule Wallaby.Experimental.Chrome do
     end
   end
 
+  @doc false
+  def execute_script_async(session_or_element, script, args \\ [], opts \\ []) do
```

Comment:
I don't really get how logging stuff works and don't use it currently, but looks the same as `execute_script` so :man_shrugging: